src/ergw_aaa_radius.erl

```diff
@@ -358,8 +363,9 @@ vendor_ituma('Wifi-Connect-Typ', #{'WLAN-Authentication-Mode' := open}, Attrs) -
     [{?'IM_Wifi_Connect_Type', <<"o">>}|Attrs];
 vendor_ituma('SSID', #{'SSID' := SSID}, Attrs) ->
     [{?'IM_SSID', SSID}|Attrs];
-vendor_ituma('Called-Station-Id', #{'SSID' := SSID, 'BSSID' := BSSID}, Attrs) ->
-    Id = lists:flatten(lists:join($;, [BSSID, SSID])),
+vendor_ituma('Called-Station-Id', #{'SSID' := SSID, 'Location-Id' := Location}, Attrs) ->
+    LocId = format_eui(<< (binary_to_integer(Location)):6/bytes >>),
```

Comment:
Judging by the data that is transmitted in `radius_SUITE.erl` the `Location` is integer. Or this wrong logic, or CT should be update